pkg/utils/log.go

```diff
@@ -50,8 +51,8 @@ func (aviLogger *AviLogger) Infof(template string, args ...interface{}) {
 	aviLogger.sugar.Infof(template, args...)
 }
 
-func (aviLogger *AviLogger) Info(args ...interface{}) {
-	aviLogger.sugar.Info(args...)
+func (aviLogger AviLogger) Info(msg string, args ...interface{}) {
+	aviLogger.sugar.Info(msg, args)
```

Comment:
Do you have to change teh Warn function below too ?

Reply:
It is not required since the warn function is not part of the klog interface.

Some of the existing log functions are changed because the klog interface expects the function in that format. After changing it, the `klog.SetLogger(utils.AviLog)` function call sets the logs from the client-go pkg in the AviLogger format.

Reply:
How will existing `Warn` `warnf` statement  work?